clients/src/main/java/org/apache/kafka/common/requests/AbstractRequest.java

```diff
@@ -75,6 +75,19 @@ public T build() {
         public abstract T build(short version);
     }
 
+    public static class NoOpRequestBuilder extends Builder<AbstractRequest> {
```

Comment:
"NoOp" doesn't seem quite right here since the builder isn't an operator.  Maybe "pass through" or "simple" ?

Reply:
delegating builder?